aiosmtpd/testing/helpers.py

```diff
@@ -26,59 +26,55 @@ def reset_connection(client):
     # };
     #
     # Is this correct for Windows/Cygwin and macOS?
-    struct_format = 'hh' if sys.platform == 'win32' else 'ii'
+    struct_format = "hh" if sys.platform == "win32" else "ii"
     l_onoff = 1
     l_linger = 0
     client.sock.setsockopt(
         socket.SOL_SOCKET,
         socket.SO_LINGER,
-        struct.pack(struct_format, l_onoff, l_linger))
+        struct.pack(struct_format, l_onoff, l_linger),
+    )
     client.close()
 
 
-# For integration with flufl.testing.
-
-def setup(testobj):
-    testobj.globs['resources'] = ExitStack()
-
-
-def teardown(testobj):
-    testobj.globs['resources'].close()
-
-
-def make_debug_loop():
-    loop = asyncio.get_event_loop()
-    loop.set_debug(True)
-    return loop
+SUPPORTED_COMMANDS_TLS: bytes = (
+    b"Supported commands: AUTH DATA EHLO HELO HELP MAIL "
+    b"NOOP QUIT RCPT RSET STARTTLS VRFY"
+)
 
+SUPPORTED_COMMANDS_NOTLS = SUPPORTED_COMMANDS_TLS.replace(b" STARTTLS", b"")
 
-def start(plugin):
-    if plugin.stderr:
-        # Turn on lots of debugging.
-        patch('aiosmtpd.smtp.make_loop', make_debug_loop).start()
-        logging.getLogger('asyncio').setLevel(logging.DEBUG)
-        logging.getLogger('mail.log').setLevel(logging.DEBUG)
-        warnings.filterwarnings('always', category=ResourceWarning)
 
+def get_server_context() -> ssl.SSLContext:
+    context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+    context.check_hostname = False
+    context.load_cert_chain(
+        resource_filename("aiosmtpd.tests.certs", "server.crt"),
+        resource_filename("aiosmtpd.tests.certs", "server.key"),
+    )
+    return context
```

Comment:
Used in several test modules, so centralized here. So in the future if we're going to change the name of the files (for example), we do it once and it trickles out to all test cases.

Reply:
Would it make sense to make this a `@pytest.fixture`? Would need to update all the test functions that use these, but...

Reply:
Well ... now that you mention it ... I don't see why not ... I'll do a quick edit later.

Killing two birds with one stone: I can also see how the Github Actions I've prepared work with a push to a PR 😉